include/packetgraph/antispoof.h

```diff
@@ -21,6 +21,9 @@
 #include <packetgraph/common.h>
 #include <packetgraph/errors.h>
 
+#define PG_ARP_MAX 100
+#define PG_NPD_MAX 100
```

Comment:
as this change the brick and not the coverage(and add public define), is should be in another patch